File: Ruby/final-value-of-variable-after-performing-operations.rb
# https://leetcode.com/problems/final-value-of-variable-after-performing-operations/

# @param {String[]} operations
# @return {Integer}
def final_value_after_operations(operations)
  neg = operations.count { |str| str.include?('--')}
  pos = operations.count { |str| str.include?('++')}
  pos - neg
end

File: Python/final-value-of-variable-after-performing-operations.py
# https://leetcode.com/problems/final-value-of-variable-after-performing-operations/

class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        count = 0
        for operation in operations:
            if operation[1] == '-':
                count -= 1
            elif operation[1] == '+':
                count += 1

        return count
